Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc class DailyForecast: NSObject {
+    let date: NSDate
+    let conditionsDescription: String
+    let highTemperature: Temperature
+    let lowTemperature: Temperature
+    
+    init(JSON: Dictionary<String, AnyObject>) {
```

Comment:
If we decide to change it into failable initializer it will require changes in the app logic. `TRWeatherUpdate` always enumerate from 1 to 4 and creates 3 `DailyForecast`s. If we start returning nil there instead of object additional safety checks will be required. I am okay for now to change this explicit unwrap into if-let but I would like to keep initializer not failable because I don't know the app logic on controllers level where RAC is. 

Let's discuss it. 

1. What if something goes wrong and view `DailyForecast` cannot be created. Do we want to display then 2 or just 1 `DailyForecast`? 
2. Can we assume that `DailyForecast` is always created and there is no case when something goes wrong? Then we can use only if-let and initializer will be always returning some objects.